hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolume.java

```diff
@@ -62,6 +62,8 @@
   private static final Logger LOG =
       LoggerFactory.getLogger(StorageVolume.class);
 
+  public static final String TMP_DIR = "tmp";
```

Comment:
Probably it makes sense to parametrize this (like hdds.datanode.replication.work.dir for example)

Reply:
This one should be related to another ticket, https://issues.apache.org/jira/browse/HDDS-6449. 
I think we can resolve this value after HDDS-6449.

Reply:
Hi @symious , does your previous container replication improvement patch cover this part?

Reply:
I can not find the definition of "tmp" now, seems being removed from StorageVolume class.

Reply:
the tmp dir defined in HDDS-6449 , in review, currently defines the tmp directory in the `HddsVolume`.  Does it need to be defined in this the underlying base class?  What was discussed in the previous patch was,

> hdds/clusterID/tmp would be the best place to put it, but we need to make sure this will not affect datanode startup on downgrade.

Reply:
@neils-dev ,  I think define tmp in StorageVolume is more generic than define it in HddsVolume, in case in future other types of Volume need tmp too.  what do you think?